### PR TITLE
feat(otb.services): add endpoints for overTheBox

### PR DESCRIPTION
**Pull request by Steffy29**
Endpoints for list of actions and their detail, and for unlink device
Ref: UXCT-269, UXCT-268

Signed-off-by: Stephanie Moallic <stephanie.moallic@corp.ovh.com>

### CODE REVIEW

File: src/api/overTheBox/overTheBox.v6.service.js [hunk 1]
```diff
@@ -109,6 +109,20 @@ angular.module('ovh-api-services').service('OvhApiOverTheBoxV6', ($resource, Ovh
...
+      isArray: true,
+    },
+    getAction: {
+      method: 'GET',
+      url: '/overTheBox/:serviceName/device/actions/:actionId',
+    },
+    unlinkDevice: {
+      method: 'DELETE',
+      url: '/overTheBox/:serviceName/device',
+      interceptor,
```
Comment (marie-j) [inline src/api/overTheBox/overTheBox.v6.service.js:124]: This looks like it could be in a `device` submodule

Comment (Steffy29) [reply src/api/overTheBox/overTheBox.v6.service.js:124]: you're right

Comment (marie-j) [reply src/api/overTheBox/overTheBox.v6.service.js:124]: Maybe the endpoints declaration could be removed in this file

File: src/api/overTheBox/overTheBox.v6.service.js [hunk 2]
```diff
@@ -109,6 +109,20 @@ angular.module('ovh-api-services').service('OvhApiOverTheBoxV6', ($resource, Ovh
       url: '/overTheBox/:serviceName/device/logs ',
       isArray: false,
     },
+    getActions: {
```
Comment (FredericEspiau) [inline src/api/overTheBox/overTheBox.v6.service.js:112]: You should delete these ones as they at in the `src/api/overTheBox/device/overTheBox-device.v6.service.js` file 🙂

**Pull request merged**